scripts/dev_settings.sh

```diff
@@ -9,7 +9,7 @@ if [ -z "$EQ_RABBITMQ_ENABLED" ]; then
 fi
 
 if [ -z "$EQ_MINIMIZE_ASSETS" ]; then
-  export EQ_MINIMIZE_ASSETS=False
+    export EQ_MINIMIZE_ASSETS=True
```

Comment:
If the settings.py defaults this to on then there is no need to also set it here.